build/package/Microsoft.DotNet.Cli.Installer.targets

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    
+    <Target Name="GenerateInstallers" 
+          DependsOnTargets="Init;Layout;GeneratePkgs;GenerateDebs;GenerateMsis" />
```

Comment:
Alright I can see that, and yeah there is nothing else coming into this file, at least in the foreseaable future. I can place this in package